packages/react/src/components/VideoPlayer/VideoPlayer.js

```diff
@@ -116,7 +121,7 @@ const VideoPlayer = ({
       </div>
       {showCaption && (
         <div className={`${prefix}--video-player__video-caption`}>
-          {videoData.name} {videoDuration}
+          {videoCaption}
```

Comment:
Do we still want the `videoDuration` appened after the custom caption

Reply:
I left it since that was in the existing caption but I will defer to design on this one

Reply:
@emyarod @ariellalgilmore functional specs says it gets appended, so it looks like that's correct: https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Video#1-caption

Reply:
thanks for checking! I will leave it as is